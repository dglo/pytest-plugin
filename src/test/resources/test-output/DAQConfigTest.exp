tsts=5
errs=2
fail=0
time=12.852
xml=false
