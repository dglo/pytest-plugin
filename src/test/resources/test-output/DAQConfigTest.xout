option -x not recognized
Usage: DAQConfigTest.py [options] [test] [...]

Options:
  -h, --help       Show this message
  -v, --verbose    Verbose output
  -q, --quiet      Minimal output

Examples:
  DAQConfigTest.py                               - run default set of tests
  DAQConfigTest.py MyTestSuite                   - run suite 'MyTestSuite'
  DAQConfigTest.py MyTestCase.testSomething      - run MyTestCase.testSomething
  DAQConfigTest.py MyTestCase                    - run all 'test*' test methods
                                               in MyTestCase

