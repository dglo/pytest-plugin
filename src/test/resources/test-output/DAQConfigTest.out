testListsSim5 (__main__.DAQConfigTest) ... ok
testListsSpsIC40IT6 (__main__.DAQConfigTest) ... ok
testLookupSim5 (__main__.DAQConfigTest) ... ERROR
testLookupSpsIC40IT6 (__main__.DAQConfigTest) ... ERROR
testReplay (__main__.DAQConfigTest) ... ok

======================================================================
ERROR: testLookupSim5 (__main__.DAQConfigTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "DAQConfigTest.py", line 74, in testLookupSim5
    self.lookup(cfg, dataList)
  File "DAQConfigTest.py", line 29, in lookup
    dom = cfg.getIDbyStringPos(data[2], data[3])
  File "/Users/dglo/prj/pdaq-dev/dash/DAQConfig.py", line 320, in getIDbyStringPos
    pos)
  File "/Users/dglo/prj/pdaq-dev/dash/DAQConfig.py", line 164, in getIDbyStringPos
    raise DOMNotInConfigException()
DOMNotInConfigException

======================================================================
ERROR: testLookupSpsIC40IT6 (__main__.DAQConfigTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "DAQConfigTest.py", line 120, in testLookupSpsIC40IT6
    self.lookup(cfg, dataList)
  File "DAQConfigTest.py", line 29, in lookup
    dom = cfg.getIDbyStringPos(data[2], data[3])
  File "/Users/dglo/prj/pdaq-dev/dash/DAQConfig.py", line 320, in getIDbyStringPos
    pos)
  File "/Users/dglo/prj/pdaq-dev/dash/DAQConfig.py", line 164, in getIDbyStringPos
    raise DOMNotInConfigException()
DOMNotInConfigException

----------------------------------------------------------------------
Ran 5 tests in 12.852s

FAILED (errors=2)
